workitem/number_sequence/workitem_number_sequence.go

```diff
@@ -1,12 +1,15 @@
-package workitem
+package numbersequence
 
 import (
+	"fmt"
+
 	uuid "github.com/satori/go.uuid"
 )
 
 // WorkItemNumberSequence the sequence for work item numbers in a space
 type WorkItemNumberSequence struct {
-	SpaceID    uuid.UUID `sql:"type:uuid" gorm:"primary_key"`
+	ID         uuid.UUID `sql:"type:uuid" gorm:"primary_key"`
```

Comment:
Why not use SpaceID as primary key?

Reply:
I thought GORM might be confused if I tried to `INSERT` a new record while the `primary` key was provided (it might run an `UPDATE` query instead)